homework/ba-nguyen/lesson-8/search-in-rotated-sorted-array-ii.ts

```diff
@@ -0,0 +1,27 @@
+function search(nums: number[], target: number): boolean {
+  let lo = 0;
+  let hi = nums.length - 1;
+
+  while (lo <= hi) {
```

Comment:
what is the runtime?

Reply:
average case: O(log(n))
worse case: O(n) (all element of the **nums** array is the same)